tests/testsuite/workspaces.rs

```diff
@@ -1074,6 +1077,7 @@ fn new_warning_with_corrupt_ws() {
         .with_stderr(
             "\
 [CREATING] binary (application) `bar` package
+[NOTE] see more `Cargo.toml` keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Could this be printed only when Cargo succeeds to create a new package?

Reply:
In this case it does succeed, its just warning about the state of the workspace.

But I can move it later in case a spurious error happens through the process.